app/components/reward.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class RewardComponent extends Component {
+    @tracked rewardPoints = '10$';
```

Comment:
change to rewardEarned

Reply:
changed

Reply:
if the quiz and training are more than 500, then how will reward points will be only 10$. It's less right, change it to some higher value

Reply:
done